.github/workflows/publish-docfx.yml

```diff
@@ -0,0 +1,37 @@
+
+name: Deploy DocFx
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
```

Comment:
See how we do concurrency in other yamls. If you add support to run (but not commit) this on PRs, then this will need to be updated.

Reply:
Updated.